modules/living_spaces_menus/living_spaces_menus.module

```diff
@@ -13,46 +13,79 @@ use Drupal\Core\Url;
  */
 function living_spaces_menus_dom_bootstrap_collapsible_menu_alter(&$build) {
   $spaces_menu_key = 'menu_link_content:436e7a5e-7eb0-4ba7-928c-b9064865f174';
-  $have_favorites = FALSE;
 
-  $query = \Drupal::entityTypeManager()->getStorage('group')->getQuery();
-  $query->addTag('living_spaces_menus_user_spaces');
-  $query->accessCheck();
-  $query->condition('status', 1, '=')
-    ->sort('label', 'ASC')
-    ->range(0, 8);
+  $user = \Drupal::entityTypeManager()
+    ->getStorage('user')
+    ->load(\Drupal::currentUser()->id());
 
-  $result = $query->execute();
+  $user_favorite_spaces = [];
+  $user_preferred_spaces = [];
+  $user_joined_spaces = [];
+  $max_total = 8;
```

Comment:
no need to create the variable if it is used only one time.

Reply:
Done.